cmake/compiler_config.cmake

```diff
@@ -24,6 +24,8 @@ set(msvc_flags
     /Zc:rvalueCast
     # enable multiprocessor builds.
     /MP
+    # enable exception handling.
+    /EHsc
```

Comment:
Full compiler support for the Standard C++ exception handling model that safely unwinds stack objects requires /EHsc (recommended). We previously had support this flag but I missed to add it to the compiler_config